scripts/tests/test_validate_docstrings.py

```diff
@@ -502,6 +533,42 @@ def no_punctuation(self):
         return "Hello world!"
 
 
+class BadOrder(object):
+
+    def wrong_section_order(self):
+        """
+        See also should not come before Returns section.
+
+        See Also
+        -------
+
+        Returns
+        -------
+        str
+            A nice greeting
+         """
+
+    def wrong_section_order1(self):
```

Comment:
I think to test all the sections we need to test every pair of contiguos section, parameters/return...